facia-tool/public/css/style.css

```diff
@@ -625,6 +625,10 @@ h1 {
     background: #FFF8E1;
 }
 
+.front-container.attention {
```

Comment:
It will be nicer to have it as a separate class. Just `.attention` should be sufficient.